tests/process_monitor/test_process_monitor.py

```diff
@@ -566,10 +597,12 @@ def test_MantarrayProcessesMonitor__hard_stops_and_joins_processes_and_logs_queu
 
     actual = mocked_logger.call_args_list[1][0][0]
     assert "Remaining items in process queues: {" in actual
-    assert expected_ok_comm_item in actual
-    assert expected_file_writer_item in actual
-    assert expected_da_item in actual
-    assert expected_server_item in actual
+    assert str(expected_file_writer_item) in actual
+    assert str(expected_da_item) in actual
+
+    process_monitor._redact_from_queue_items(expected_file_writer_item)
```

Comment:
We want to avoid calling private functions/methods in tests directly, so in this test you'll need to take a similar approach to the tests above